modules/servers/upCloudVps/lib/configOptions.php

```diff
@@ -39,10 +42,10 @@ private function createCustomConfigurableOptions($product)
   {
     $currencyId = Capsule::table('tblcurrencies')->where('default', '1')->first()->id;
     $currencyCode = Capsule::table('tblcurrencies')->where('default', '1')->first()->code;
-          $groupId = Capsule::table('tblproductconfiggroups')->where('name', 'Configurable options for UpCloud - Custom Plans')->first()->id;
+          $groupId = Capsule::table('tblproductconfiggroups')->where('name', 'Configurable options for UpCloud - custom plans')->first()->id;

```

Comment:
The few changes like these are here in order to make the option sets look consistent wrt capitalization etc. New product setup is required for the new option sets to take effect, so take care of these while at it.

Reply:
Makes sense to do this now for the first release